packages/monaco/src/browser/style/index.css

```diff
@@ -63,3 +63,7 @@
     border-bottom-color: var(--theia-accent-color2);
 	position: relative;
 }
+
+.monaco-editor .parameter-hints-widget>.wrapper {
```

Comment:
[minor] can you adjust the spacing so it becomes:

```css
.monaco-editor .parameter-hints-widget > .wrapper
```

Reply:
I did it as you said